DESCRIPTION

```diff
@@ -20,7 +24,9 @@ Encoding: UTF-8
 LazyData: true
 Imports:
   arrow,
+  DBI,
   dplyr,
```

Comment:
Add `dbplyr`

Reply:
It _should_ be okay with just dbplyr + duckdb + DBI.
RMD check indicates it's okay so far.

Reply:
Oh, weird. I installed all these dependencies, loaded the package, and got an error saying `dbplyr` still needed to be installed. (I would have assumed it to be a dependency of `duckdb` and/or `DBI`.) Perhaps an isolated issue.